Pipfile

```diff
@@ -11,7 +11,15 @@ uwsgitop = ">=0.11"
 uwsgi-tools = ">=1.1.1"
 lxml = ">=3.5.0,<4.2.6"
 invenio-app-ils = {editable = true,git = "https://github.com/inveniosoftware/invenio-app-ils.git",ref = "master",extras = [ "lorem" ]}
+invenio-migrator = {editable = true,version = "==1.0.0a10"}
+cds-books = {editable = true,path = "."}
+invenio-db = {editable = true,version = "*"}
+invenio-files-rest = "*"
+invenio-records-files = "*"
+cds-dojson = {editable = true,git = "https://github.com/CERNDocumentServer/cds-dojson",ref = "books"}
```

Comment:
I would really prefer to depend on a release, we might forget.
Or we really have to remember to fix it afterwards!